### PR TITLE
Adds MIX Destroy Element, its Parser and Serializer.

### PR DESCRIPTION
**Pull request by tarun018**

### CODE REVIEW

File: Swiften/Elements/MIXDestroy.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#pragma once
+
+#include <memory>
+
+#include <Swiften/Base/API.h>
+#include <Swiften/Base/Override.h>
```
Comment (tfar) [inline Swiften/Elements/MIXDestroy.h]: This include isn't used. Please remove.

**Comment by tfar** [comment]
Looks good. Could you please squash them into a single commit.

**Comment by tarun018** [comment]
Squashed into single commit.

**Comment by tfar** [comment]
Thank you. This PR has been submitted and merged upstream.

**Pull request closed without merging**